packages/web-components/src/components/ic-menu/ic-menu.spec.tsx

```diff
@@ -0,0 +1,963 @@
+import { newSpecPage } from "@stencil/core/testing";
+import { waitForTimeout } from "../../testspec.setup";
+import { Menu } from "./ic-menu";
+import { Select } from "../ic-select/ic-select";
```

Comment:
I think we were hoping to test the ic-menu in isolation i.e. without needing to import\include ic-select & ic-search-bar